pallets/phala/src/stakepool.rs

```diff
@@ -541,11 +560,101 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Manually assign rewards to pools by miningSwitchOrigin for fixing issue 763
```

Comment:
This is the change from earlier PR. Please merge / rebase to the latest master

Reply:
done